tests/go/GoReturnsBearTest.py

```diff
@@ -29,6 +29,5 @@
 
 GoReturnsBearTest = verify_local_bear(
```

Comment:
In GoReturnsBear documentation already shows goreturns instead of gofmt. There were no changes needed for this. 

This is the state of current documentation:
    """
    Proposes corrections of Go code using ``goreturns``.
    """